modules/tinymce/src/core/main/ts/api/html/DomParser.ts

```diff
@@ -63,6 +63,8 @@ export interface DomParserSettings {
   forced_root_block_attrs?: Record<string, string>;
   inline_styles?: boolean;
   preserve_cdata?: boolean;
+  /**
+   * @deprecated filter was moved to tinymce.dom.serializer and will be removed in the next major release */
```

Comment:
Filter hasn't really been moved for now, just added to Serializer. 
```suggestion
   * @deprecated Remove trailing <br> tags functionality has been added to tinymce.dom.Serializer and option will be removed in the next major release */
```